lib/screens/landing/landing_screen.dart

```diff
@@ -1,14 +1,55 @@
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
+import 'package:survey/gen/assets.gen.dart';
 import 'package:survey/screens/screen.dart';
 
-class LandingScreen extends StatelessWidget {
+class LandingScreen extends StatefulWidget {
+  @override
+  _LandingScreenState createState() => _LandingScreenState();
+}
+
+class _LandingScreenState extends State<LandingScreen>
+    with TickerProviderStateMixin {
```

Comment:
if there is only one animation, we can use `SingleTicketProviderStateMixin` instead 🙏